README.md

```diff
@@ -28,6 +28,12 @@ For webpack, add a [resolve.alias](http://webpack.github.io/docs/configuration.h
 
 Browser support: IE8+ and everything else.
 
+## Installation
+
+```
+npm i xhr
```

Comment:
Change the `i` to `install`? I use the shorthand myself, but in my experience more developers are familiar with the long name than the shorthand. I don't think we want anyone to think, "what will running this command do?" – it'd be better to appeal to a wider audience for the install imo.